src/sql/workbench/contrib/notebook/browser/notebookExplorer/notebookSearchResultsView.ts

```diff
@@ -0,0 +1,378 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as DOM from 'vs/base/browser/dom';
```

Comment:
There seems to be a lot of duplicated classes in this file. Can we not re-use the existing ones from vs code?

Reply:
I removed the classes and interfaces that can be reused from search and added only the ones that's needed for notebook search.